# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# Result
# side of the handle "left" or "right"
string handle_side
# status of the door "open" or "closed"
string door_status
# if it's closed and we have the position of the handle
geometry_msgs/PoseStamped door_handle
# if it's open and we have the position of the door centroid
geometry_msgs/PoseStamped door_position
